test/module/iroha-cli/client_test.cpp

```diff
@@ -162,6 +165,125 @@ TEST_F(ClientTest, SendTxWhenStatelessInvalid) {
       "                    \"peer_key\": "
       "\"2323232323232323232323232323232323232323232323232323232323232323\"\n"
       "                }]}";
-  std::cout << "Sending  json transaction to Iroha" << std::endl;
+
   ASSERT_EQ(client.sendTx(json_tx), iroha_cli::CliClient::NOT_VALID);
 }
+
+TEST_F(ClientTest, SendQueryWhenInvalidJson) {
```

Comment:
This test a little bit confusing, because in cli no reason to test public server API. I guess this test need to move in integration test folder.